core/domain/user_services.py

```diff
@@ -359,7 +359,7 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
-                role=feconf.ROLE_ID_ADMIN,
+                role=feconf.ROLE_ID_EXPLORATION_EDITOR,
```

Comment:
Sorry, I missed this. The system committer does need to be able to commit to private explorations/collections, so we are in a bit of a bind here. We may need to either allow them admin privileges or invent a new role just for them that allows editing stuff but not sending email. I'm agnostic between these two options, as long as it's done cleanly and safely.